build/common/init_command.sh

```diff
@@ -41,6 +41,7 @@ elif [ "$MODULE_NAME" = 'openvpn' ]; then
     supervisord --nodaemon --configuration supervisord.conf
 elif [ "$MODULE_NAME" = 'nginx' ]; then
     rm -rf /etc/nginx/conf.d/default.conf
+    pip install tldextract
```

Comment:
let's install this in the Dockerfile of the base image please, add it to the requirements.txt and pin the version.